frontend/src/metabase/query_builder/actions/query-updates.ts

```diff
@@ -0,0 +1,20 @@
+import type { Dispatch, GetState } from "metabase-types/store";
+
+import * as MetabaseLib from "metabase-lib/v2";
```

Comment:
FE folks, do you have any naming preferences here? `MetabaseLib` feels a bit too massive; how about just `Lib` or `ML`? Other ideas are welcome